sdk/translation/Azure.AI.Translation.Document/src/DocumentTranslationClient.cs

```diff
@@ -219,11 +219,11 @@ public virtual async Task<DocumentTranslationOperation> StartTranslationAsync(Do
         /// </summary>
         /// <param name="filter">Options to use when filtering result.</param>
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
-        public virtual Pageable<TranslationStatus> GetAllTranslationStatuses(TranslationFilter filter = default, CancellationToken cancellationToken = default)
```

Comment:
consider updating the docstring too, so we don't say `results for all submitted translation...` and instead say `results for submitted translations`. 
Or we could say `Get the status results for all submitted translation operations according to the values of the filters"

Reply:
same comment will apply for the Documents methods

Reply:
Yea sure. I went with the former suggestion because I felt like it is clearer and more straightforward. I noticed that we also won't be calling them filters anymore so maybe I can change it to : 

`Get the status results for all submitted translation operations according to the options provided`

but this may imply that options **need** to be provided, so I just chose to simply remove the word all. What do you think?

Reply:
sounds good!